example/src/App.tsx

```diff
@@ -136,6 +136,12 @@ class App extends React.Component<AppProps, AppState> {
     }).catch((error) => {
       this.logOutput(`logConversion: error ${JSON.stringify(error)}\n`);
     });
+    Radar.logTermination();
```

Comment:
I'm not seeing a lifecycle listener for this — if you can't find one, maybe comment this out and explain that this would be added in a listener for app termination events on iOS (we automatically handle this on Android)